.github/workflows/test.yaml

```diff
@@ -63,4 +66,20 @@ jobs:
 
       - run: patchright install --no-shell chromium
 
-      - run: pytest tests/${{ matrix.test }}.py
+      - name: Install Playwright and Chromium
+        run: |
+          uv pip install playwright
+          uv run playwright install chromium
+          uv run patchright install chrome
```

Comment:
```suggestion
```
none of this is needed we already intall it directly above this block.

especially dont install `chrome`, that's old-headless which is no longer used by any code path.